backend/src/course/course.ts

```diff
@@ -0,0 +1,86 @@
+import { getExternalRanking } from "../rating/ranking";
+import { getVideoDetailByIds, getVideosByQuery } from "../rating/youtube";
+import Parse from "parse/node";
+import { IResource } from "../types/resource";
+import { IWeightedYoutubeVideo } from "../types/youtube";
+
+const VIDEOS_PER_QUERY = 100;
+
+export const createCourse = (
+  name: string,
+  user: Parse.Object<Parse.Attributes>
+) => {
+  const Course: Parse.Object = new Parse.Object("Course");
+  Course.set("name", name);
+  Course.set("user", user);
+  return Course;
+};
+
+export const createResource = (resource: IResource) => {
+  const Resource: Parse.Object = new Parse.Object("Resource");
+
+  Object.keys(resource).forEach((resourceAttribute) => {
+    Resource.set(resourceAttribute, resource[resourceAttribute]);
+  });
+
+  return Resource;
+};
+
+export const saveResources = async (
+  resources: IWeightedYoutubeVideo[],
+  course: Parse.Object<Parse.Attributes>,
+  level: 1 | 2,
```

Comment:
You can use an enum here, so that we can now what level 1 and 2 mean:

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
I didn't know Enums even existed on Typescript! Thanks for letting me know!!